cdi/4.0/_index.md

```diff
@@ -27,6 +27,7 @@ Jakarta Contexts Dependency Injection specifies a means for obtaining objects in
 * [Weld 5.0.0.CR2](https://weld.cdi-spec.org/download/)
 
 =======
```

Comment:
The [website](https://deploy-preview-439--jakartaee-specifications.netlify.app/specifications/cdi/4.0/) is still not displayed well. Maybe just remove the line 29.

Reply:
The preview looks ok now.